src/Services/Air/Air.js

```diff
@@ -81,6 +82,32 @@ module.exports = (settings) => {
       return service.getTicket(options);
     },
 
+    getPNRByTicketNumber(options) {
+      const terminal = createTerminalService(settings);
+      const memo = {};
+      return terminal.executeCommand(`*TE/${options.ticketNumber}`)
+        .then(
+          (response) => {
+            memo.response = response;
+          }
+        )
+        .then(
+          () => terminal.closeSession()
+        )
+        .then(
+          () => {
+            try {
+              return memo.response.match(/RLOC [^\s]{2} ([^\s]{6})/)[1];
+            } catch (err) {
+              throw new AirRuntimeError.PnrParseError(memo.response);
```

Comment:
In general it is for not throwing

TypeError: Cannot read property '1' of null